git-xlsx-textconv.go

```diff
@@ -27,7 +27,7 @@ func main() {
 			}
 			cels := make([]string, len(row.Cells))
 			for i, cell := range row.Cells {
-				s := cell.String()
+				s, _ := cell.String()
```

Comment:
@k-ta-yamada @tokuhirom What is the idea behind this change and how does this even work for you? 😳 
For me (stable Go 1.6.2 on OS X 10.10), this PR does nothing but introduce a compile error.

> […]`git-xlsx-textconv.go:30: assignment count mismatch: 2 = 1`

@k-ta-yamada Would you mind to elaborate?